multipart/src/main.rs

```diff
@@ -57,7 +58,9 @@ pub fn handle_multipart_item(
     }
 }
 
-pub fn upload(req: HttpRequest) -> FutureResponse<HttpResponse> {
+pub fn upload(req: HttpRequest<AppState>) -> FutureResponse<HttpResponse> {
+    req.state().counter.set(req.state().counter.get() + 1);
+    println!("{:?}",  req.state().counter.get());
```

Comment:
I feel like there is not much need to print just counter

Well maybe you could just print something `Upload number: {}`?